controllers/servers/csi/exception_handler.py

```diff
@@ -16,7 +16,8 @@
     array_errors.HostNotFoundError: grpc.StatusCode.NOT_FOUND,
     array_errors.PermissionDeniedError: grpc.StatusCode.PERMISSION_DENIED,
     array_errors.ObjectIsStillInUseError: grpc.StatusCode.FAILED_PRECONDITION,
-    array_errors.CredentialsError: grpc.StatusCode.UNAUTHENTICATED
+    array_errors.CredentialsError: grpc.StatusCode.UNAUTHENTICATED,
+    array_errors.ObjectAlreadyExistError: grpc.StatusCode.ALREADY_EXISTS,
```

Comment:
Naming suggestion: ObjectAlreadyExists sounds like the same object is there and we should just ignore it in case it was an idempotent create request. The way it is used rather looks like ObjectParameterConflictError, meaning that we are trying to create an incompatible object with the same name.

Reply:
it's following the Spec naming, this is not visible by the user, the user will get the grpc status and the [description](https://github.com/IBM/ibm-block-csi-driver/blob/159a88d7a5ee0225cc632f9280783656032cc4f4/controllers/array_action/messages.py#L89)

Reply:
ack